group_vars/all/main.yml

```diff
@@ -105,6 +117,10 @@ openshift_routes:
   - openshift/edxapp/route/cms.yml
   - openshift/edxapp/route/lms.yml
   - openshift/richie/route/richie.yml
+openshift_routes_prefixes:
+  - previous
+  - current
+  - next
 openshift_services:
```

Comment:
This YAML should not validate since you have duplicated variables. This variable should be named `openshift_svcs` to avoid any confusion with what we call a "service" (_i.e._ `svc + dc`)

Reply:
Done